README.md

```diff
@@ -23,7 +23,30 @@ When called the method should print "Would you take this rose, `person`, in exch
 Demonstrate calling the method, passing in "young prince" as the argument.
 
 ```rb
-# Your code goes here...
+require "pry"
+
+class BeautyAndTheBeast
+
+  attr_accessor :person
+
+  def initialize ([person])
+    @person = person
+
+    def set_name_to(some_string)
+      @person = some_string
+    end
+
+    def offer_rose
+      puts "Would you take this rose #{@person} in exchange for giving an old beggar woman shelter from the bitter cold?"
+    end
+
+  end
+@young_prince.offer_rose
+end
+
+binding.pry
+
+puts "end of test"
```

Comment:
You don't need to define a class for this question. All you need to define is a method called `offer_rose`. So something like this...

```rb
# method definition
def offer_rose person
  puts "Would you take this rose, #{person}, in exchange for giving an old beggar woman shelter from the bitter cold?"
end

# call method
offer_rose "Adrian"
```